src/packages.builds

```diff
@@ -10,7 +16,7 @@
     <Project Include="Native\pkg\**\*.builds">
       <AdditionalProperties>$(AdditionalProperties);SkipCreatePackageOnMissingFiles=true</AdditionalProperties>
     </Project>
-    <Project Include="*\pkg\*.builds">
+    <Project Include="*\pkg\*.builds" Condition="'$(FilterToOSGroup)' == 'Windows_NT'">
```

Comment:
It is true that it feels a ltlle weird.
I talked to @jhendrixMSFT offline and build-packages will have a property to just build the native packages, so the default is still to build all packages